cmd/cluster/openstack/create.go

```diff
@@ -45,15 +47,17 @@ func bindCoreOptions(opts *RawCreateOptions, flags *pflag.FlagSet) {
 	flags.StringVar(&opts.OpenStackExternalNetworkID, "openstack-external-network-id", opts.OpenStackExternalNetworkID, "ID of the OpenStack external network (optional)")
 	flags.StringVar(&opts.OpenStackIngressFloatingIP, "openstack-ingress-floating-ip", opts.OpenStackIngressFloatingIP, "An available floating IP in your OpenStack cluster that will be associated with the OpenShift ingress port (optional)")
 	flags.StringSliceVar(&opts.OpenStackDNSNameservers, "openstack-dns-nameservers", opts.OpenStackDNSNameservers, "List of DNS nameservers to use for the cluster (optional)")
+	flags.Var(&opts.OpenStackImageRetentionPolicy, "openstack-image-retention-policy", "OpenStack Glance Image retention policy. Valid values are 'Orphan' and 'Prune'. By default images are pruned. (optional)")
```

Comment:
if the option is required, should this not mention the optional in the message?

Reply:
yeah I got confused in the discussion elsewhere where we said we could make it required with a default but after re-reading https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#defaulting : 

> With configuration APIs, we typically default fields within the controller and not within the API.

So better to make it optional.